### PR TITLE
DOCS OBSDOCS-847 Logging 5.8.4 Release Notes

### PR DESCRIPTION
**Pull request by shreyasiddhartha**
Change type: Doc update; Logging Z-Stream Release Notes - 5.8.4
Doc JIRA: https://issues.redhat.com/browse/OBSDOCS-847

Fix Version: 4.12+

Doc Preview: https://72505--ocpdocs-pr.netlify.app/openshift-enterprise/latest/logging/logging_release_notes/logging-5-8-release-notes#logging-release-notes-5-8-4_logging-5-8-release-notes 

SME Review: @periklis 
QE Review: @anpingli 
Peer Review: @nalhadef

### CODE REVIEW

**Review by anpingli — COMMENTED** [review]
/lgtm

**Review by periklis — COMMENTED** [review]
LGTM

File: modules/logging-release-notes-5-8-4.adoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+* Before this update, multiple `ClusterLogForwarders` defining the same input receiver name had their service endlessly reconciled because of changing `ownerReferences` on one service. With this update, each receiver input will have its own service named with the convention of `<CLF.Name>-<input.Name>`. (link:https://issues.redhat.com/browse/LOG-5009[LOG-5009])
+
+* Before this update, the `ClusterLogForwarder` did not report errors when forwarding logs to cloudwatch without a secret. With this update, the following error message appears when forwarding logs to cloudwatch without a secret: `secret must be provided for cloudwatch output`. (link:https://issues.redhat.com/browse/LOG-5021[LOG-5021])
+
+* Before this update, the `log_forwarder_input_info` included `application`, `infrastructure`, and `audit` input metric points. With this update, `http` is also added as a metric point. (link:https://issues.redhat.com/browse/LOG-5043[LOG-5043])
```
Comment (nalhadef) [inline modules/logging-release-notes-5-8-4.adoc:18]: Is https already a metric point?

**Comment by nalhadef** [comment]
Looks good. Just one comment.

File: modules/logging-release-notes-5-8-4.adoc (hunk 1 above)
Comment (shreyasiddhartha) [reply modules/logging-release-notes-5-8-4.adoc:18]: As mentioned in the bug (https://issues.redhat.com/browse/LOG-5043) and PR (https://github.com/openshift/cluster-logging-operator/pull/2357) `http` is added as a metric point.

**Pull request merged**